ssl/ssl_locl.h

```diff
@@ -1314,7 +1314,12 @@ typedef struct ssl3_state_st {
      * that the server selected once the ServerHello has been processed.
      */
     unsigned char *alpn_selected;
-    unsigned alpn_selected_len;
+    unsigned int alpn_selected_len;
```

Comment:
Mostly to fix "unsigned"; as before, size_t was used for a specific API purpose. I can check to see how alpn_selected_len is used, and could switch it.